tests/unit/utils/test_path.py

```diff
@@ -267,11 +266,80 @@ def test_existing_binary_in_windows_pathext(self):
             # Let's patch os.environ to provide a custom PATH variable
             with patch.dict(os.environ, {'PATH': os.sep + 'bin',
                                          'PATHEXT': '.COM;.EXE;.BAT;.CMD;.VBS;'
-                                         '.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY'}):
+                                                    '.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY'}):
                 # Let's also patch is_windows to return True
                 with patch('salt.utils.platform.is_windows', lambda: True):
                     with patch('os.path.isfile', lambda x: True):
                         self.assertEqual(
                             salt.utils.path.which('this-binary-exists-under-windows'),
                             os.path.join(os.sep + 'bin', 'this-binary-exists-under-windows.CMD')
                         )
+
+
+@skipIf(NO_MOCK, NO_MOCK_REASON)
+class TestPath(TestCase):
+    def setUp(self):
+        self.dir_exist = os.path.join(os.path.dirname(__file__), 'dir_exist')
```

Comment:
It might be better to use temporary files and directories generated from the `tempfile` module: https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory

Reply:
Done.